fl4health/parameter_exchange/parameter_packer.py

```diff
@@ -71,3 +73,47 @@ def unpack_parameters(self, packed_parameters: NDArrays) -> Tuple[NDArrays, List
         model_parameters = packed_parameters[:split_size]
         param_names = packed_parameters[split_size:][0].tolist()
         return model_parameters, param_names
+
+
+class SparseCooParameterPacker(ParameterPacker[Tuple[NDArrays, NDArrays, List[str]]]):
+    """
+    This parameter packer is responsible for selecting an arbitrary set of parameters
+    and then representing them in the sparse COO tensor format, which requires knowing
+    the indices of the parameters within the tensor to which they belong,
+    the shape of that tensor, and also the name of it.
+    """
+
+    def pack_parameters(
+        self, model_parameters: NDArrays, additional_parameters: Tuple[NDArrays, NDArrays, List[str]]
+    ) -> NDArrays:
+        parameter_indices, tensor_shapes, tensor_names = additional_parameters
+        return model_parameters + parameter_indices + tensor_shapes + [np.array(tensor_names)]
+
+    def unpack_parameters(self, packed_parameters: NDArrays) -> Tuple[NDArrays, Tuple[NDArrays, NDArrays, List[str]]]:
+        assert len(packed_parameters) % 3 == 1
+        split_size = (len(packed_parameters) - 1) // 3
+        model_parameters = packed_parameters[:split_size]
+        parameter_indices = packed_parameters[split_size : (2 * split_size)]
+        tensor_shapes = packed_parameters[(2 * split_size) : (3 * split_size)]
+        tensor_names = packed_parameters[(3 * split_size) :][0].tolist()
+        return model_parameters, (parameter_indices, tensor_shapes, tensor_names)
+
+    def extract_coo_info_from_dense(self, x: Tensor) -> Tuple[NDArray, NDArray, NDArray]:
```

Comment:
Pretty minor, but I think we can make this an `@staticmethod`, as it isn't specific to a particular instance.